config.ini-sample

```diff
@@ -1,2 +1,4 @@
 [MapQuest]
 key=your-api-key-goes-here
+[Directory]
+dir=%Y-%m-%b
```

Comment:
We might want to be more explicit here about this being the format for the date. It might make sense to include a `location` attribute as well that would let people customize how the location folder is named.

```
[Directory]
date=%Y-%m-%b
location=%city-%state-%country
```